chain/bitcoind_events_test.go

```diff
@@ -214,6 +217,46 @@ func testNotifySpentMempool(t *testing.T, miner *rpctest.Harness,
 	}
 }
 
+// testLookupInputMempoolSpend tests that LookupInputMempoolSpend returns the
+// correct tx hash and whether the input has been spent in the mempool.
+func testLookupInputMempoolSpend(t *testing.T, miner *rpctest.Harness,
```

Comment:
Can we refactor these tests to also exercise the `btcd` version? Or is this already covered by the `btcd` integration tests themselves?

Reply:
Yeah that's covered there, but I definitely agree we should add a new test `btcd_test.go` here to check the RPC client behavior, maybe something to do when we decide to refactor the `chain` package.